src/CommonGLPI.php

```diff
@@ -854,20 +854,29 @@ public function showTabsContent($options = [])
             }
         }
 
+        $cleaned_options = $options;
+        if (isset($cleaned_options['id'])) {
+            unset($cleaned_options['id']);
+        }
+        if (isset($cleaned_options['stock_image'])) {
+            unset($cleaned_options['stock_image']);
+        }
```

Comment:
Unset can take multiple arguments and will not trigger any warning if array keys are not set.
```suggestion
        unset($cleaned_options['id'], $cleaned_options['stock_image']);
```